scrapy/downloadermiddlewares/httpcompression.py

```diff
@@ -17,6 +18,9 @@
     from typing_extensions import Self
 
 ACCEPTED_ENCODINGS: List[bytes] = [b"gzip", b"deflate"]
+logger = logging.getLogger(__name__)
+
+ACCEPTED_ENCODINGS = [b"gzip", b"deflate"]
```

Comment:
This looks like a merge error.

Reply:
Yup. I'm working on some more changes. Many tests got broken, fixed that already. Polishing next commit now.